Update sql-data-warehouse-tables-index.md (#118122)
Azure/azure-content: articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-index.md

WITH ( HEAP );
```

> [!NOTE]

> > If you frequently perform INSERT, UPDATE, or DELETE operations on a heap table, it is advisable to include table rebuilding in your maintenance schedule by using ALTER TABLE command (i.e., ALTER TABLE [SchemaName].[TableName]  REBUILD). This practice contributes to reduced fragmentation, resulting in improved performance during read operations.

Reviewer: ```suggestion
> If you frequently perform `INSERT`, `UPDATE`, or `DELETE` operations on a heap table, it is advisable to include table rebuilding in your maintenance schedule by using `ALTER TABLE` command. For example, `ALTER TABLE [SchemaName].[TableName] REBUILD`. This practice contributes to reduced fragmentation, resulting in improved performance during read operations.
```

Reviewer: Formatting. @dialmoth please accept commit. Also, please edit the file's `ms.date` up on line 6. Thanks!
